packages/block-editor/src/components/inserter/block-patterns-tab/pattern-category-previews.js

```diff
@@ -163,23 +170,39 @@ export function PatternCategoryPreviews( {
 					</Text>
 				) }
 			</VStack>
-
-			{ currentCategoryPatterns.length > 0 && (
-				<BlockPatternsList
-					ref={ scrollContainerRef }
-					shownPatterns={ pagingProps.categoryPatternsAsyncList }
-					blockPatterns={ pagingProps.categoryPatterns }
-					onClickPattern={ onClickPattern }
-					onHover={ onHover }
-					label={ category.label }
-					orientation="vertical"
-					category={ category.name }
-					isDraggable
-					showTitlesAsTooltip={ showTitlesAsTooltip }
-					patternFilter={ patternSourceFilter }
-					pagingProps={ pagingProps }
-				/>
-			) }
+			<VStack spacing={ 4 }>
+				{ currentCategoryPatterns.length > 0 && (
+					<>
+						{ isZoomOutMode && (
```

Comment:
Do we only want the prompt in zoom out mode?

Reply:
If this looks like a useful information addition we should add it in general to the patterns tab.